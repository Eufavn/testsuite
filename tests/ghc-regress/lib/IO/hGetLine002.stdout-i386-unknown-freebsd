this line doesn't end with a newline
end of file
Action: hGetLine
Handle: {loc=<stdin>,type=readable,binary=False,buffering=block (1024)}
File: <stdin>
