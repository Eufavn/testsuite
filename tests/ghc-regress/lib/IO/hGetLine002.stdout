this line doesn't end with a newline
end of file
Action: hGetLine
Handle: {loc=<stdin>,type=readable,binary=False,buffering=line}
File: <stdin>
